yt/testing.py

```diff
@@ -373,6 +373,9 @@ def fake_hexahedral_ds():
     dist = np.sum(_coordinates ** 2, 1)
     node_data[("connect1", "test")] = dist[_connectivity - 1]
 
+    for field in fields:
```

Comment:
you'll need to add `from more_itertools import always_iterable`
```suggestion
    for field in always_iterable(fields):
```